docs/sources/flow/reference/components/prometheus.exporter.mongodb.md

```diff
@@ -29,7 +29,7 @@ Omitted fields take their default values.
 Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
 `mongodb_uri` | `string` | MongoDB node connection URI. | | yes
-`direct_connect` | `boolean` | Whether or not a direct connect should be made. Direct connections are not valid if multiple hosts are specified or an SRV URI is used. | true | no
```

Comment:
This is fixing a typo in our docs identified when testing the defaults in the static converter tests. It may even be that we should default to true but it is a potentially breaking change to mess with defaults so I didn't do it in this PR.

Reply:
Good catch! May look like a breaking change to users, but it's actually a documentation bug.